src/components/activefilters/activefilters.js

```diff
@@ -161,15 +161,16 @@ export default class ActiveFilters {
   }
 
   handleFilterClick(e) {
-    const type = e.target.getAttribute('data-type');
-    const name = e.target.getAttribute('data-name');
-    const value = e.target.getAttribute('data-value');
-    const container = e.target.getAttribute('data-container');
-    const confFields = e.target.getAttribute('data-conf-fields')
-      ? e.target.getAttribute('data-conf-fields').split(',')
+    const targetButton = e.currentTarget;
```

Comment:
Issue: While the change from e.target to e.currentTarget is correct for button clicks, the handler is directly attached to buttons. If users press Space/Enter on the button, this should work, but there's no verification in the code review of how event listeners are attached.

  Recommendation: Verify in activefilters.js that event listeners are attached like:
  button.addEventListener('click', this.handleFilterClick);

  And ensure keyboard events (Space/Enter) are also handled, though native buttons should handle this automatically.

Reply:
Such eventlistener does exist (even before this PR)

https://github.com/AddSearch/search-ui/blob/3109afc4ba6e9ec8a8acdb00b32ba6b40da3fe49/src/components/activefilters/activefilters.js#L151-L163